client/src/components/FeedbackForm/FeedbackButton.module.scss

```
@@ -0,0 +1,6 @@
+.button {
+  font-size: 0.75em;
+  &:hover {
+    color: #9d85ef;
+  }
+}
```

Can you make this bold? I believe adding a `font-weight: 500` should do the trick.
```suggestion
.button {
  font-size: 0.75em;
  font-weight: 500;
  &:hover {
    color: #9d85ef;
  }
}
```